pages/overview/what-is-jsonschema.md

```diff
@@ -6,36 +6,287 @@ title: What is JSON Schema?
-
-JSON Schema is highly extensible and can be tailored to fit your needs. You can create custom keywords, formats, and validation rules to suit your own requirements. The JSON Schema community has a wealth of tools and resources available across many programming languages to help you create, validate, and integrate your schemas.
+The current version of JSON Schema is [2020-12](../draft/2020-12/release-notes.md), which represents the latest advancements and have expended capabilities as compared with the previous version `draft-04`, `draft-06`, `draft-07`. We encourage everyone to adopt the latest version whenever possible to take advantage of all the advancements and benefits of JSON Schema.
 
+For more information regarding JSON Schema history, you can refer to [this article](https://modern-json-schema.com/what-is-modern-json-schema) by **Henry Andrews**.
 
 ## How it works
 
 Using JSON Schema, you can define rules and constraints that JSON data should adhere to. When your JSON documents adhere to these constraints, it becomes easier to exchange structured data between applications because the data follows a consistent pattern.
 
-You begin by defining the rules in a JSON Schema document using various [keywords](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-validation-keywords-for-any). For example, you can use `type` to specify what types of data are valid for a given key. You can specify either a single type, like `string`, or an array of possible types, like `string`, `number`, or `integer`. You can also enforce formats using regular expressions with the `pattern` keyword. There are countless ways to describe specific data types and formats that your data must follow.
+### What is a JSON Document
+
+Before we get into JSON Schema and how it can help us, let's first understand what exactly is a JSON document.
+
+* A JSON document represents a piece of data that follows the syntax and structure defined by the JSON format. It is a collection of key-value pairs, arrays, and nested objects. 
+* JSON documents are used to store and transfer data between systems and applications.
+
+Taking an example of a JSON document representing a customer order,
+```
+{
+  "order_id": "123456",
+  "customer_name": "John Doe",
+  "items": [
+    {
+      "product_id": "P001",
+      "name": "T-shirt",
+      "quantity": 2,
+      "price": 19.99
+    },
+    {
+      "product_id": "P002",
+      "name": "Jeans",
+      "quantity": 1,
+      "price": 49.99
+    }
+  ],
+  "total_amount": 89.97,
+  "status": "pending"
+}
+```
+
+* The above code snippet includes attributes such as the *order ID*, *customer name*, *items ordered* (an array of objects with product details), *shipping address*, *total amount*, and *status* of the order.
+
+* This JSON document provides a structured representation of an order, making it easy to exchange, store, or process the order information in various applications or systems.
+
+
+### Without JSON Schema
```

Comment:
Instead of the current sentence "Without JSON Schema" I'll suggest something like "The challenge"